reactive-architecture/apps/prices/push_image.sh

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env bash
 
-REGISTRY="quay.io"
+REGISTRY="registry.ocp4.example.com:8443"
 IMAGE="${REGISTRY}/redhattraining/do378-reactive-architecture-prices"
 
-podman login quay.io
+podman login registry.ocp4.example.com:8443
```

Comment:
I think I initially forgot to use the REGISTRY var here :rofl: 
```suggestion
podman login ${REGISTRY}
```